finance/grc.fraudDetection_web/README.md

```diff
@@ -0,0 +1,143 @@
+# fraudDetection_web
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework in order to let us present results with GraphQL.
+In this project, we use TypeDB to represent a queryable database of relevant fraud detection data about some transactions between Cardholders and Companies.
+
+## Introduction
+
+We have an imaginary transaction dataset and our application implements some very basic functions:
+
+- Search for Bank
+- Search for Cardholder
+- Search for Company
+- Search for Transaction 
+
+We are demonstrating sub-typing, powerful rules and rule combination in our schema. 
+####
+We can also see how easy it is to create complex queries using query composition.
+####
+We are using the DAO design pattern, we have the following components on which our design depends:
+
+- The model which is transferred from one layer to the other.
+- The interfaces which provide a flexible design.
+- The interface implementation which is a concrete implementation of the persistence logic.
+
+The query composition can be observed in all DAOs.
+```java
+String getQueryStr = "match " + TX_MATCH + CardholderDao.CARDHOLDER_MATCH + BankDao.BANK_MATCH + MerchantDao.MERCHANT_MATCH;
+```
+
+## Running the application in dev mode
+
+1. Checkout this repository: `git clone https://github.com/vaticle/typedb-examples && cd typedb-examples/grc.fraudDetection_web`.
+2. Start the [TypeDB Server](http://docs.vaticle.com/docs/running-typedb/install-and-run#start-the-typedb-server). Check that it's listening to address: `0.0.0.0:1729`.
+3. You can run your application in dev mode that enables live coding using:
+```shell script
+./mvnw compile quarkus:dev
```

Comment:
I haven't been able to run this locally. Let's confirm that this is a problem with my particular setup rather than an issue that someone might have with running it fresh.